src/replicache.test.ts

```diff
@@ -224,7 +224,9 @@ async function replicacheForTesting(
     diffServerAuth,
     diffServerURL,
     name,
-    repmInvoke: invoke,
```

Comment:
I intentionally used a function and not an object with invoke... Doesn't really matter since this code will go away at some point (or only be used by tests)... also more flexible.